accordion/index.html

```diff
@@ -0,0 +1,126 @@
+        }
+    </style>
+</head>
+
+<body>
+    <h2>Accordion Example</h2>
+    <section>
+        <div class="accordion">Section 1</div>
+        <div class="segment">
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec dictum ipsum, sed maximus massa.
+                Curabitur eleifend
+                lorem non posuere tristique. Maecenas vel porta augue. Phasellus lobortis ante id gravida egestas.
+                Pellentesque
+                facilisis ut odio in rhoncus. Duis ut sollicitudin massa. Aenean non nisi est. Nullam fermentum, sapien
+                posuere luctus
+                imperdiet, enim nisl vehicula dui, sit amet bibendum lectus risus in odio. Praesent sed sem tempor, pretium
+                orci in,
+                condimentum neque. Cras a magna eget erat tempus tincidunt ut quis enim. Praesent efficitur sollicitudin
+                lectus ac
+                vehicula
+            </p>
+        </div>
+    </section>
+    <section>
+        <div class="accordion">Section 2</div>
+        <div class="segment">
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec dictum ipsum, sed maximus massa.
+                Curabitur eleifend
+                lorem non posuere tristique. Maecenas vel porta augue. Phasellus lobortis ante id gravida egestas.
+                Pellentesque
+                facilisis ut odio in rhoncus. Duis ut sollicitudin massa. Aenean non nisi est. Nullam fermentum, sapien
+                posuere luctus
+                imperdiet, enim nisl vehicula dui, sit amet bibendum lectus risus in odio. Praesent sed sem tempor, pretium
+                orci in,
+                condimentum neque. Cras a magna eget erat tempus tincidunt ut quis enim. Praesent efficitur sollicitudin
+                lectus ac
+                vehicula
+            </p>
+        </div>
+    </section>
+    <section>
+        <div class="accordion">Section 3</div>
+        <div class="segment">
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec dictum ipsum, sed maximus massa.
+                Curabitur eleifend
+                lorem non posuere tristique. Maecenas vel porta augue. Phasellus lobortis ante id gravida egestas.
+                Pellentesque
+                facilisis ut odio in rhoncus. Duis ut sollicitudin massa. Aenean non nisi est. Nullam fermentum, sapien
+                posuere luctus
+                imperdiet, enim nisl vehicula dui, sit amet bibendum lectus risus in odio. Praesent sed sem tempor, pretium
+                orci in,
+                condimentum neque. Cras a magna eget erat tempus tincidunt ut quis enim. Praesent efficitur sollicitudin
+                lectus ac
+                vehicula
+            </p>
+        </div>
+    </section>
+
+    <script>
+        const accordion = document.getElementsByClassName("accordion");
```

Comment:
Please move the JS and CSS in a separate files and include the link back to the HTML